qtpy/__init__.py

```diff
@@ -278,6 +278,12 @@ def __init__(self, *, missing_package=None, **superclass_kwargs):
 if API in PYSIDE6_API:
     try:
         from PySide6 import __version__ as PYSIDE_VERSION  # analysis:ignore
+
+        if PYSIDE_VERSION == "6.8.0":
+            print(
+                "A known critical bug in PySide6 6.8.0 will cause your application to crash "
+                "see https://github.com/spyder-ide/qtpy/issues/494",
+            )
```

Comment:
Shouldn't we add a `return` below this line to avoid the segfault?

Reply:
Actually, that's not a good idea because it'll be generated in another module. So, ignore my comment.

Reply:
Yeah. I mean. I’m not trying to be fancy here. Just a bit of damage control.

Reply:
We shouldn’t overthink these things

Reply:
Agreed. I'll release 2.4.2 with your fix during the weekend or on Monday at most.